src/NTPClientLib.cpp

```diff
@@ -199,6 +199,7 @@ time_t NTPClient::getTime () {
 }
 #elif NETWORK_TYPE == NETWORK_ESP8266 || NETWORK_TYPE == NETWORK_ESP32
 void NTPClient::s_dnsFound (const char *name, const ip_addr_t *ipaddr, void *callback_arg) {
+    (void)name;
```

Comment:
Why do you need this?

Reply:
This is to avoid a compiler warning indicating that a parameter is unused.
It is visible when full & verbose logs are enabled in the arduino IDE preferences panel.
This idiom explicitly tells to the compiler to stay silent because we already know about that fact and that's on purpose.

Here's the warning:
```
libraries/NtpClient/src/NTPClientLib.cpp: In static member function 'static void NTPClient::s_dnsFound(const char*, const ip_addr_t*, void*)':
libraries/NtpClient/src/NTPClientLib.cpp:201:41: warning: unused parameter 'name' [-Wunused-parameter]
  201 | void NTPClient::s_dnsFound (const char *name, const ip_addr_t *ipaddr, void *callback_arg) {
      |                             ~~~~~~~~~~~~^~~~
```